crypto/key_manager.go

```diff
@@ -22,11 +22,11 @@ type KeyManager interface {
 	Signer() (crypto.Signer, error)
 	// GetPublicKey returns the public key previously loaded. Potentially nil if the key
 	// manager requires initialization and this has not been done
```

Comment:
Is it in golang style to not bother talk about error returns, because they are so pervasive?